proposals/0025/README.md

```diff
@@ -0,0 +1,55 @@
+---
+id: 0025
+title: Deprecate OSIE
+status: discussion
+authors: Thomas Stromberg
+---
+
+## Summary
+
+Deprecate and archive the tinkerbell/osie repository
+
+## Background
+
+OSIE is the Operating System Installation Environment. OSIE has been in use by
+Equinix Metal since before Tinkerbell was created. OSIE contains three major
+components, only one of which has been in use by the Tinkerbell OSS community:
+
+* A live Linux environment, built with Alpine Linux
```

Comment:
maybe we call out that this is what Hook replaces? similar to the 2 below. `deprecated by Hook`?

Reply:
Done - please take another look.